tools/malloc_size_of/Cargo.toml

```diff
@@ -10,44 +10,30 @@ path = "lib.rs"
 
 [features]
 servo = [
-    "crossbeam-channel",
     "hyper",
     "hyper_serde",
-    "keyboard-types",
     "serde",
     "serde_bytes",
     "string_cache",
     "time",
     "url",
-    "webrender_api",
-    "xml5ever",
-    "content-security-policy",
     "uuid",
-    "accountable-refcell",
 ]
 
 [dependencies]
-accountable-refcell = { version = "0.2.0", optional = true }
 app_units = "0.7"
-content-security-policy = {version = "0.3.0", features = ["serde"], optional = true}
 crossbeam-channel = { version = "0.3", optional = true }
-cssparser = "0.27"
 euclid = "0.20"
-hashglobe = { path = "../hashglobe" }
 hyper = { version = "0.12", optional = true }
 hyper_serde = { version = "0.11", optional = true }
 keyboard-types = {version = "0.4.3", optional = true}
```

Comment:
We should probably rip out all this stuff as well, no point in building it. Or I guess we won't build it because we won't use the `servo` feature? If so, fine.

Reply:
Yeah, more ripping out is possible.  I intentionally left in things I thought we might use (we do use hyper) or could conceivably use (string caches might be neat?), but it also looks like I left some stuff in Cargo.toml that I removed from lib.rs.  I'll do a more thorough pass shortly now that it's all working.

Reply:
Okay, I ripped out everything except for `git2` in Cargo.toml.  I'm pretty confident in my ability to copy-and-paste back anything we end up needing like the limited "hyper" support!